README.md

```diff
@@ -100,13 +102,10 @@ good fit for Alacritty. This means you won't find things like tabs or splits
 (which are best left to a window manager or [terminal multiplexer][tmux]) nor
 niceties like a GUI config editor.
 
-## IRC
-
-Alacritty discussions can be found in `#alacritty` on Libera.Chat.
-
 ## License
 
 Alacritty is released under the [Apache License, Version 2.0].
 
 [Apache License, Version 2.0]: https://github.com/alacritty/alacritty/blob/master/LICENSE-APACHE
 [tmux]: https://github.com/tmux/tmux
+[`#alacritty`]: https://web.libera.chat/gamja/?channels=#alacritty
```

Comment:
Can we move this under the paragraph itself? I don't like moving the definitions to the end of the file it makes it rather difficult to find where things are actually used.

Reply:
Should we do the same wrt tmux?

Reply:
Probably. I'm not blocking the PR on it but it would make sense to have it consistent.